tests/CI/run-integration-test/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM mbedos/mbed-os-env:latest
+RUN pip3 install virtualenv
```

Comment:
install latest mbed-tools

Reply:
Done: f34b9a8c57c80f5dc1f5119f1748c072510726c4

Reply:
Marking this as unresolved. Why do we need to install mbed-tools here? It's packaged with the base image.